processes/fulfillURLManifest.py

```diff
@@ -46,18 +46,17 @@ def getManifests(self, startTimeStamp=None):
 
         batches = self.load_batches(self.prefix, self.s3Bucket)
         if startTimeStamp:
-            filtered_batches = batches.search(f"Contents[?to_string(LastModified) > '\"{startTimeStamp}\"'].Key")
-            for batch in filtered_batches:
-                for c in batch['Contents']:
-                    currKey = c['Key']
-                    metadataObject = self.getObjectFromBucket(currKey, self.s3Bucket)
-                    self.update_manifest(metadataObject, self.s3Bucket, currKey)
+            #Using JMESPath to extract keys from the JSON batches
```

Comment:
Hmm what is the purpose of this comment? What is JMESPath?

Reply:
JMESPath is a query language for JSON and the comment is here so we know what type of query was made here since it's not used anywhere else in the codebase from my observations.